tests/syntax-tests/create_highlighted_versions.py

```diff
@@ -31,16 +43,24 @@ def create_highlighted_versions(output_basepath):
             env.pop("BAT_TABS", None)
             env["COLORTERM"] = "truecolor"  # make sure to output 24bit colors
 
-            bat_output = subprocess.check_output(
-                ["bat"] + BAT_OPTIONS + [source], stderr=subprocess.PIPE, env=env,
-            )
-
-            source_dirname = path.basename(path.dirname(source))
+            source_dirpath = path.dirname(source)
+            source_dirname = path.basename(source_dirpath)
             source_filename = path.basename(source)
 
-            if source_filename == "LICENSE.md":
+            if source_filename in SKIP_FILENAMES:
                 continue
 
+            options = copy.deepcopy(BAT_OPTIONS)
```

Comment:
For simple lists, you can also use `options = BAT_OPTIONS.copy()` without using a separate module.